wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -1116,8 +1130,9 @@ class KotlinGenerator private constructor(
       if (empty) return null
     }
     return PropertySpec.builder(fieldName, optionsType.typeName)
-            .initializer(initializer)
-            .build()
+        .jvmField()
+        .initializer("\n%L", initializer)
+        .build()
```

Comment:
wasn't it only when we would have to generate java interop code?

Reply:
yes, we thought it should work in interop mode